plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -2,13 +2,332 @@
+						{{i18n("remote-config.percentage")}}\
+					</th>\
+				</tr>\
+			</thead>\
+			<tbody>\
+                <tr><td><div class="cly-vue-remote-config-percentages-breakdown__sequence bu-py-1 bu-px-2">1</div></td><td><div class="bu-is-align-items-center	bu-is-flex cly-vue-remote-config-percentages-breakdown__data bu-p-1 cly-vue-remote-config-percentages-breakdown__default-value"><span class="bu-ml-2 bu-mr-3">{{i18n("remote-config.default-value")}}</span><span class="cly-vue-remote-config-percentages-breakdown__default-value__value bu-py-1 bu-px-2">{{defaultValue.value}}</span></div></td><td>{{defaultValue.percentage}} {{i18n("remote-config.percent.of.total")}}</td></tr>\
+				<tr v-for="(condition, i) in conditions" :key="i">\
+                    <td><div class="cly-vue-remote-config-percentages-breakdown__sequence bu-py-1 bu-px-2">{{i+2}}</div>\
+                    </td>\
+					     <td><div class="bu-is-align-items-center	bu-is-flex cly-vue-remote-config-percentages-breakdown__data bu-p-1 cly-vue-remote-config-percentages-breakdown__condition" :style="{backgroundColor: condition.color}"><span><i class="ion ion-git" style="transform:rotate(90deg)"></i></span><span class="bu-ml-2 bu-mr-3">{{condition.name}}</span><span class="cly-vue-remote-config-percentages-breakdown__condition__value bu-py-1 bu-px-2">{{condition.value}}</span></div></td>\
+					<td>\
+                    {{condition.percentage}} {{i18n("remote-config.percent.of.total")}}\
+					</td>\
+				</tr>\
+			</tbody>\
+		</table>',
+        props: {
+            parameter: {
+                type: Object,
+                default: function() {
+                    return {};
+                }
+            }
+        },
+        computed: {
+            conditions: function() {
+                var conditions = [];
+                var conditionsMap = this.$store.getters["countlyRemoteConfig/parameters/conditionsMap"];
+                var self = this;
+                if (this.parameter.conditions.length > 0) {
+                    this.parameter.conditions.forEach(function(condition) {
+                        var conditionProperties = conditionsMap[condition.condition_id];
+                        var ob = {
+                            color: self.getColor(conditionProperties.condition_color),
+                            percentage: condition.c ? ((condition.c) / self.totalConditions).toFixed(2) : 0,
+                            name: conditionProperties.condition_name,
+                            value: condition.value
+                        };
+                        conditions.push(ob);
+                    });
+                }
+                return conditions;
+            },
+            totalConditions: function() {
+                var total = 0;
+                if (this.parameter.conditions.length > 0) {
+                    this.parameter.conditions.forEach(function(condition) {
+                        total = total + condition.c ? condition.c : 0;
```

Comment:
What about adding the count of requests that received default value.
Shouldn't we add that too the total count in order to find the percentages?